lib/tasks/seed.rake

```diff
@@ -0,0 +1,95 @@
+require 'factory_girl'
+
+namespace :seed do
```

Comment:
Oh, I like that idea and I fully agree the current name is too generic. Just something I hadn't gotten around to.